Framework/Beamline/inc/MantidBeamline/EigenTest.h

```diff
@@ -0,0 +1,13 @@
+#ifndef MANTID_EIGENTEST_H
```

Comment:
I was planning to remove `EigenTest.h` and `EigenTest.cpp` completely before removing 'In Progress'. It was just to test the build servers can actually build something using `Eigen`.